content/articles/user-input-and-while-loops-in-python/index.md

```diff
@@ -0,0 +1,198 @@
+As a programmer, many programs that you'll write will solve an end user's problem. You will always want to get some data from the user. This article will teach us how to accept user input and how to use while loops. While loops enable your programs to remain executing when certain conditions remain true.
+
+### Prerequisites
+1. Python installed on your computer.
+2. Basic Python knowledge.
+
+### How the input() function works
+The `input()` function halts your program and waits the user to key in some data. When Python receives the user's input, store it in the variable that you choose to work with.
```

Comment:
Rewrite "The `input()` function halts your program and waits the user to key in some data." to "The `input()` function halts the execution of a program and waits for the user to key in some data."